python/MDSplus/tests/_UnitTest.py

```diff
@@ -274,7 +275,7 @@ def tearDownClass(cls):
                 shutil.rmtree(cls.tmpdir)
     def tearDown(self):
         gc.collect()
-        if self.inThread: return
+        if not self.is_win or self.inThread: return
```

Comment:
What's the purpose of this?

Reply:
under windows the file locking is different and for a proper test traceback we need to make sure we release the tree files even in case of an error. otherwise it may cause an error in a subsequent test of the same module. under linux we dont have to and we should not because python seems to double __del__ for some versions. valgrind would complain but its an issue with pyhton